test/test_index_string.cpp

```diff
@@ -1687,6 +1687,27 @@ TEST(StringIndex_Fuzzy)
 }
 
 
+namespace {
+
+// results returned by the index should be in ascending row order
+// this requirement is assumed by the query system which runs find_gte
+// and this will return wrong results unless the results are ordered
+auto check_result_order = [](const IntegerColumn& results, TestContext& test_context) {
```

Comment:
Originally I had this in a single test and I was capturing `test_context`, then I moved it out to an anonymous namespace to be reused from multiple tests and I had to make the context a parameter. Should I change it to a function?